swap_meet/clothing.py

```diff
@@ -0,0 +1,8 @@
+from swap_meet.item import Item 
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
```

Comment:
```suggestion
    def __init__(self, condition = 0):
```
class Clothing will always have "clothing" as its category, so we don't need to give the user the ability to change that as an argument